.env-example

```diff
@@ -1,2 +1,3 @@
 PORT=8080
 MODE=replay
+STOREURL=https://my-dev-store-759854325.store.bcdev
```

Comment:
It should be a placeholder, not your own VM store URL.

Reply:
updated